src/abstract/ExecutorAware.sol

```diff
@@ -24,61 +33,37 @@ abstract contract ExecutorAware {
    * @param _executor Address of the `MessageExecutor` contract
    */
   constructor(address _executor) {
-    require(_executor != address(0), "executor-not-zero-address");
-    trustedExecutor = _executor;
+    _setTrustedExecutor(_executor);
   }
 
-  /* ============ External Functions ============ */
+  /* ============ Public Functions ============ */
 
   /**
    * @notice Check which executor this contract trust.
    * @param _executor Address to check
    */
   function isTrustedExecutor(address _executor) public view returns (bool) {
-    return _executor == trustedExecutor;
+    return _executor == _trustedExecutor;
   }
 
-  /* ============ Internal Functions ============ */
-
   /**
-   * @notice Retrieve messageId from message data.
-   * @return _msgDataMessageId ID uniquely identifying the message that was executed
+   * @notice Getter for the internal trusted executor.
+   * @return The trusted executor address
    */
-  function _messageId() internal pure returns (bytes32 _msgDataMessageId) {
-    _msgDataMessageId;
-
-    if (msg.data.length >= 84) {
-      assembly {
-        _msgDataMessageId := calldataload(sub(calldatasize(), 84))
-      }
-    }
+  function trustedExecutor() public view returns (address) {
```

Comment:
you can replace this by making trustedExecutor public

Reply:
Fixed by: [b42e6df](https://github.com/GenerationSoftware/erc5164-interfaces/pull/1/commits/b42e6df6353a5aa6eec1ba9454d27ef57382bda0)